Sources/TuistAutomation/XcodeBuild/XcodeBuildController.swift

```diff
@@ -61,7 +62,11 @@ public final class XcodeBuildController: XcodeBuildControlling {
         // Destination
         switch destination {
         case let .device(udid):
-            command.append(contentsOf: ["-destination", "id=\(udid)"])
+            var value = ["id=\(udid)"]
+            if rosetta {
+                value += ["arch=x86_64"]
+            }
```

Comment:
Is the id of the rosetta and non-rosetta simulator the same? Otherwise you need to move this logic to `SimulatorController`

Reply:
Based on the output of `xcrun simctl list -j devices available`, there is no separate rosetta simulators.
I think they would alway use the same IDs.